README.md

```diff
@@ -44,20 +44,66 @@ For more information, see the [documentation for `Pkg.jl`](https://pkgdocs.julia
+
+```julia
+using Oceananigans
+using Oceananigans.Units
+using Dates, CFTime
+import ClimaOcean
+
+arch = GPU()
+grid = LatitudeLongitudeGrid(arch,
+                             size = (1440, 560, 10),
+                             halo = (7, 7, 7),
+                             longitude = (0, 360),
+                             latitude = (-70, 70),
+                             z = (-3000, 0))
+
+bathymetry = ClimaOcean.regrid_bathymetry(grid) # builds gridded bathymetry based on ETOPO1
+grid = ImmersedBoundaryGrid(grid, GridFittedBottom(bathymetry))
+
+# Build an ocean simulation initialized to the ECCO state estimate on Jan 1, 1993
+ocean = ClimaOcean.ocean_simulation(grid)
+date  = DateTimeProlepticGregorian(1993, 1, 1)
+set!(ocean.model, T = ClimaOcean.ECCOMetadata(:temperature; date),
+                  S = ClimaOcean.ECCOMetadata(:salinity; date))
+
+# Build and run an OceanSeaIceModel (with no sea ice component) forced by JRA55 reanalysis
+atmosphere = ClimaOcean.JRA55_prescribed_atmosphere(arch)
+coupled_model = ClimaOcean.OceanSeaIceModel(ocean; atmosphere)
+simulation = Simulation(coupled_model, Δt=10minutes, stop_time=30days)
+run!(simulation)
+```
+
+ClimaOcean's core abstractions are `ocean_simulation` and `ClimaOcean.OceanSeaIceModel`:
+
+* `ocean_simulation` configures an Oceananigans model for realistic simulations including temperature and salinity, the TEOS-10 equation of state, boundary conditions to store computed air-sea fluxes, the automatically-calibrated turbulence closure `CATKEVerticalDiffusivity`, and the [`WENOVectorInvariant` advection scheme](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2023MS004130) for mesoscale-turbulence-resolving simulations.
 
-In summary, if you're interested in realistic, hydrostatic regional or global simulations you may find `ClimaOcean` useful.
-Otherwise, you can stick with [Oceananigans](https://github.com/CliMA/Oceananigans.jl).
+* `OceanSeaIceModel` provides a framework for coupled modeling with an ocean component, prescribed atmosphere, and optionally, a sea ice component.
+`OceanSeaIceModel` computes air-sea, air-ice, and ice-ocean fluxes of momentum, heat, and freshwater.
 
+In addition to these core abstractions `ClimaOcean` provides convenience features for wrangling datasets of bathymetry, ocean temperature, salinity, and velocity fields, and prescribed atmospheric states.
     
+`ClimaOcean` is built on top of `Oceananigans` and `ClimaSeaIce`, so it's important that `ClimaOcean` users become proficient with [`Oceananigans`](https://github.com/CliMA/Oceananigans.jl).
+Note that `ClimaOcean` is currently focused on hydrostatic modeling with `Oceananigans`' `HydrostaticFreeSurfaceModel`.
+
+This simulation achieves approximately 8 years per day of wall time on an Nvidia H100.
+Since, for example, `ocean.model` is an Oceananigans.HydrostaticFreeSurfaceModel`, we can leverage all of Oceananigans features in our scripts.
```

Comment:
```suggestion
Since, for example, `ocean.model` is an `Oceananigans.HydrostaticFreeSurfaceModel`, we can leverage all of Oceananigans features in our scripts.
```